src/stan/io/reader.hpp

```diff
@@ -34,33 +34,35 @@ namespace io {
 template <typename T>
 class reader {
  private:
-  std::vector<T> &data_r_;
-  std::vector<int> &data_i_;
+  Eigen::Map<Eigen::Matrix<T, -1, 1>, Eigen::Aligned8, Eigen::Stride<0, 0>>
```

Comment:
Why require alignment here?

Reply:
I guess I don't know if Aligned8 is restrictive or not. Also the stride requirement?

Reply:
Umm, yeah I think we can remove both of these